saitodisse/nextjs-opinionated
# [add-react-hook-forms] Form Example: react hook form +  zod

![image](https://user-images.githubusercontent.com/155953/121959789-9dd9e700-cd3b-11eb-9132-608df8f74177.png)

### how to test

- add some email, first button should be enabled if email is valid
- send a **valid email** with **first button**, you should get a **success message**
- send a **valid email** with **second button**, you should get a **success message**
- send a **invalid email** with **second button**, you should get an **json error message**
